doc/help.rst

```diff
@@ -2,38 +2,98 @@
+
+|
+
+Automatic help can be individually enabled for both the :guilabel:`Editor` and the :guilabel:`Console` under :menuselection:`Preferences --> Help --> Automatic Connections`, and turned on and off dynamically via the :guilabel:`lock` icon in the top right corner of the :guilabel:`Help` pane.
 
 ========================
 Understanding help modes
 ========================
 
 You can use the options menu (:guilabel:`Gear` icon) in the top right of the :guilabel:`Help` pane to toggle the help display mode.
 
-:guilabel:`Rich Text` mode renders the object's docstrings with ``Sphinx``:
+.. image:: images/help/help-options-menu.png
+   :width: 600
+   :align: center
+   :alt: Spyder Help pane, displaying the options menu
+
+|
+
+:guilabel:`Rich Text` mode renders the object's docstrings with ``Sphinx``, :guilabel:`Plain Text` mode displays the docstring without formatting while :guilabel:`Show Source` displays the docstring inline with the code for the selected object, or any Python portion of it (if the object is not pure Python). The latter can be useful when docstrings are not available or insufficient to document the object.
+
+|
+
+.. raw:: html
 
-.. image:: images/help/help_standard.png
-   :alt: Spyder Help panel, displaying rich text docs for the DataFrame class
+	<iframe src="https://giphy.com/embed/cOEYQdiZCqtEpb2CsQ" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/cOEYQdiZCqtEpb2CsQ">via GIPHY</a></p>
 
 |
 
-:guilabel:`Plain Text` mode displays the docstring without formatting:
 
-.. image:: images/help/help_plain.png
-   :alt: Spyder Help panel, displaying plain text docs for the DataFrame class
+========================
+Getting help by hovering
+========================
+
+You can also get summary help for objects by hovering over them in the :guilabel:`Editor`.
+Clicking the hover popup will open the full documentation in the :guilabel:`Help` pane.
+
+.. image:: images/help/help-hover.png
+   :width: 600
+   :align: center
+   :alt: Editor pane showing help of an object from hover 
 
 |
 
-:guilabel:`Show Source` displays the docstring inline with the code for the selected object, or any Python portion of it (if the object is not pure Python).
-This can be useful when docstrings are not available or insufficient to document the object:
 
-.. image:: images/help/help_source_code.png
-   :alt: Help panel showing part of the docstring and source code for DataFrame
+
+========================
+Control automatic import
+========================
+
+When you get help in the :guilabel:`IPython Console` for an object that has not been previously imported, it is automatically loaded in the :doc:`internalconsole` so that documentation can be shown when available.
+This can be disabled in the :guilabel:`Help` pane's top-right options menu so that only documentation from imported objects is displayed.
 
 |
```

Comment:
```suggestion
```

This was left over from removing an image. When moving or removing images, remember to also take care of any related markeup and line breaks as the current doc is littered with these issues I've had to suggest fixes for.